devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy

```diff
@@ -47,7 +47,7 @@ metadata {
 	}
 
 	preferences {
-		input "tempOffset", "number", title: "Temperature offset", description: "Select how many degrees to adjust the temperature.", range: "*..*", displayDuringSetup: false
+		input "tempOffset", "number", title: "Temperature offset", description: "Enter how many degrees to adjust the temperature.", range: "-100..100", displayDuringSetup: false
```

Comment:
The `description` field is set by the UX team and the existing string has already had all the translations done so it cannot be changed at this time. Please only change the `range`.